library/common/extensions/filters/http/platform_bridge/filter.cc

```diff
@@ -23,6 +23,9 @@ namespace HttpFilters {
 namespace PlatformBridge {
 
 namespace {
+
+constexpr auto SlowCallbackWarningThreshold = std::chrono::seconds(1);
```

Comment:
+1 to starting at 1 second here. Ideally we roll this out at scale and see that it's rarely hit, if ever, and we can reduce it accordingly. Ideally pt2 is that we find a good way to make this configurable.